### PR TITLE
Troy  1.1.1

### PR DESCRIPTION
**Pull request by SmituSingh**
Hi Troy! I'm creating a pull request for the changes.

### CODE REVIEW

File: docs/_pages/configuring.markdown [hunk 1]
```diff
@@ -10,101 +10,77 @@ description: How to configure TRIRIGA Indoor Maps for your IBM TRIRIGA solution.
...
-triBuilding | Esri Building Default Rotation (triEsriRotationNU) | New field. See installation instructions for adding.  Depends on the shape of the building and the most common screen format people will view it in.  If the building is wider East to West it probably makes sense to rotate it 90 degrees.  Experiment with it to find the best value for the most common medium.
+triBuilding | GIS Latitude (triGisLatitudeNU) | This is an existing field.  It can be easily obtained by googling the address and looking in the URL. You may have to tweak the link to get to the preferred position.
+triBuilding | GIS Longitude (triGisLongitudeNU) | This is an existing field.  It can be easily obtained by googling the address and looking in the URL. You may have to tweak the link to get to the preferred position.
+triBuilding | Esri Map ID (triEsriMapIdTX) | this is a new field. For adding, refer to the installation instructions. See the following instructions for configuring. (Ex. 403b6012276b4339af152e9c3319a4b1)
```
Comment (troyibm) [inline docs/_pages/configuring.markdown]: replace `this` with `This`.

File: docs/_pages/configuring.markdown [hunk 2]
```diff
@@ -10,101 +10,77 @@ description: How to configure TRIRIGA Indoor Maps for your IBM TRIRIGA solution.
...
 
 
 
 ## Obtaining Esri Map ID
 
-1. Open a web browser.
-1. Navigate to your Esri installation's Portal e.g. `https://host.domain/portal`
-1. Click the `Sign In` on the top right of the landing page.
+1. Navigate to your Esri installation's portal, e.g. `https://host.domain/portal`.
+2. Click `Sign In` on top right of the landing page.
```
Comment (troyibm) [inline docs/_pages/configuring.markdown]: Markdown automatically numbers the numbered list correctly so you don't have to.  That's one feature of markdown that makes it more editor-friendly.  Replace all the numbers with 1.

File: docs/_pages/installation.markdown [hunk 3]
```diff
@@ -18,5 +18,5 @@ sidebar:
 Follow these instructions to [install and configure Esri software]({{ site.url }}{{ site.baseurl }}/install-tririga) to generate and serve indoor maps from CAD data.
 
 ### Install and configure TRIRIGA software
-
+=======
```
Comment (troyibm) [inline docs/_pages/installation.markdown:22]: The single line with a single hyphen is important for Jekyll to generate the HTML from a markdown file.  Put back the -.

**Review by troyibm — CHANGES_REQUESTED** [review]

File: docs/_pages/configuring.markdown (hunk 2 above)
Comment (troyibm) [reply docs/_pages/configuring.markdown]: Fix this in all the other files too.

**Comment by SmituSingh** [comment]
Have made the corrections.

**Pull request closed without merging**